tests/filters/test_canny.py

```diff
@@ -300,7 +300,7 @@ def test_module(self, device, dtype):
     @pytest.mark.skipif(torch_version() in {"2.0.0", "2.0.1"}, reason="Not working on 2.0")
     def test_dynamo(self, batch_size, kernel_size, device, dtype, torch_optimizer):
         if (
-            torch_version() in {"2.1.1", "2.1.2"}
+            torch_version() in {"2.1.1", "2.1.2", "2.2.0"}
```

Comment:
Should this use the decorator instead to skip versions ? It’s in the two places now

Reply:
in the decorator, we cannot check the dtype... and I keep it waiting for it to work in a future release